src/main/kotlin/no/nav/syfo/brev/narmesteleder/NarmesteLederService.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.syfo.brev.narmesteleder
+
+import no.nav.syfo.client.narmesteleder.NarmesteLederClient
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.domain.Virksomhetsnummer
+
+class NarmesteLederService(
+    private val narmesteLederClient: NarmesteLederClient
+) {
+
+    suspend fun getVirksomhetsnummer(
+        arbeidstakerIdent: PersonIdentNumber,
+        narmesteLederIdent: PersonIdentNumber,
+        callId: String
+    ): Virksomhetsnummer? {
+        val narmesteLedere = narmesteLederClient.narmesteLedere(arbeidstakerIdent, callId)
+        val narmesteLeder = narmesteLedere.find { nl -> nl.narmesteLederFnr == narmesteLederIdent.value }
+
+        return if (narmesteLeder != null) Virksomhetsnummer(narmesteLeder.orgnummer) else null
+    }
```

Comment:
Om jeg leser riktig så kan dette blir det feil i 2 scenarioer:
1. Om  `narmesteLederClient.narmesteLedere` henter alle ledere(aktive og tidligere ledere) så vil man få tilgang til møter på en person man ikke er leder til fordi man har vært leder en gang tidligere(gitt riktig virksomhetsnummer).
2.  Om man er registert som aktiv NL for en arbeidstaker på 2 eller flere virksomhetsnumre vil NL kun få ut møter som er knyttet til 1 virksomhetsnummer selv om man egentlig skulle hatt flere.

Hadde vært en fordel å lage tester for disse tilfellene.